conda_smithy/configure_feedstock.py

```diff
@@ -1060,6 +1059,64 @@ def render_azure(jinja_env, forge_config, forge_dir):
     )
 
 
+def _drone_specific_setup(jinja_env, forge_config, forge_dir, platform):
+    platform_templates = {
+        "linux": [
+            "build_steps.sh.tmpl",
+        ],
+        "osx": [],
+        "win": [],
+    }
+    template_files = platform_templates.get(platform, [])
+
+    # Explicitly add in a newline character to ensure that jinja templating doesn't do something stupid
+    build_setup = "run_conda_forge_build_setup\n"
+
+    if platform == "linux":
+        yum_build_setup = generate_yum_requirements(forge_dir)
+        if yum_build_setup:
+            forge_config['yum_build_setup'] = yum_build_setup
+
+    forge_config["build_setup"] = build_setup
+    forge_config["docker"]["interactive"] = False
+
+    # Override the Docker image here so we don't need to set it in the template
+    forge_config["docker"]["image"] = "condaforge/linux-anvil-aarch64"
```

Comment:
hmm you probably want to be able to set this in the template.  For example, you may wish to swap out the docker container and that shouldn't require a new conda-smithy release